.changeset/funny-forks-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@propeldata/ui-kit': patch
+---
+
+Add all Query Hook links to the overview page
```

Comment:
You can remove the changeset, it's unnecessary for this PR as it only affects storybook but not the UI-Kit library